main.cpp

```diff
@@ -108,15 +108,38 @@ struct CarWash
     You'll need to insert the Person struct from the video in the space below.
  */
 
+struct LeftFoot 
+{
+    int step;
+};
+struct RightFoot
+{ 
+    int step;
+};
+
```

Comment:
You should be using 1 struct for this class.  The `leftFoot`, rightFoot are two instances of the same class.
Also, always initialize your member variables
- [ ] revise